sdks/go/pkg/beam/io/databaseio/writer_test.go

```diff
@@ -0,0 +1,33 @@
+package databaseio
+
+import "testing"
+
+func TestValueTemplateGenerator(t *testing.T) {
```

Comment:
Since you have multiple test cases for this function, we usually do it in a table-format as written in the example: https://github.com/apache/beam/blob/d615b624e9ff211e857d026d541c4d56fd18e2d3/sdks/go/pkg/beam/io/rtrackers/offsetrange/offsetrange_test.go#L30

Just to keep it consistent, could you format it like the one in link above.

Reply:
Thanks for the review!

I've updated the tests according to the reference you mentioned. I also refactored `colsCount` to `columnCount` to match other code.

Also added a missing license (I think it was failing the check).